actors/builtin/reward/reward_calc.py

```diff
@@ -26,6 +30,9 @@ def epochs_in_year() -> Decimal:
 def q128(val, round) -> str:
     return str((Q128 * val).to_integral_value(round))
 
+def atto(val) -> str:
+    return str((10**18 * val).to_integral_value(decimal.ROUND_DOWN))
```

Comment:
@davidad I'm not finding any rhyme or reason to our rounding. Can we just safely round down (i.e., truncate)?

Reply:
Fine with me. All our fixed-point arithmetic does truncation too, we just aim to have enough precision that it doesn't matter.